src/webroot/js/cake-core.js

```diff
@@ -0,0 +1,25 @@
+//$(document).ready(function () {
+  var elements = document.getElementsByClassName('cake-core-postLink');
+
+  var postLinkFunction = function (event) {
+    var formName = this.getAttribute("data-cake-core-form");
+    var confirmMessage = this.getAttribute('data-cake-core-confirm');
+    if (confirmMessage !== null && confirmMessage !== 'undefined') {
+      if (confirm(confirmMessage)) {
+        document[formName].submit();
+        return false;
+      }
+      else {
```

Comment:
I think we are using 4 spaces for JS files, as well.
Also, no else needed when returning early.